generators/server/templates/src/main/java/package/config/_CacheConfiguration.java

```diff
@@ -277,4 +291,60 @@ public SessionRegistry sessionRegistry() {
     }
     <%_ } _%>
     <%_ } _%>
+
+    <%_ if (hibernateCache == 'infinispan' || clusteredHttpSession == 'infinispan') { _%>
+    private final Logger log = LoggerFactory.getLogger(CacheConfiguration.class);
+
+    /* for customizing session cookie key; default is 'session'
+    @Bean
+    public org.springframework.session.web.http.CookieSerializer cookieSerializer() {
```

Comment:
we dont normally qualify classes like this. Wouldnt the IDE import it when the code is un-commented? anyway I'm not sure of it. @jdubois what do you think?